share/models/base.py

```diff
@@ -52,7 +52,7 @@ class ShareObjectMeta(ModelBase):
     share_attrs = {
         'change': lambda: models.OneToOneField(Change, related_name='affected_%(class)s', editable=False, on_delete=DATABASE_CASCADE),
         'date_modified': lambda: models.DateTimeField(auto_now=True, editable=False, db_index=True, help_text=_('The date this record was modified by SHARE.')),
-        'date_created': lambda: models.DateTimeField(auto_now_add=True, editable=False, help_text=_('The date of ingress to SHARE.')),
+        'date_created': lambda: models.DateTimeField(auto_now_add=True, editable=False, db_index=True, help_text=_('The date of ingress to SHARE.')),
```

Comment:
Once we merge https://github.com/CenterForOpenScience/SHARE/pull/645 could you use it to add the new index?